histogram.js

```diff
@@ -153,9 +145,16 @@ function core(opt) {
     //.width("100%")
     .margin({ top: 5, right: 20, bottom: 20, left: 55 })
     .sizeRatio(.75)
-    .tooltip(function(d) {
-      return d[1];
-    });
+
+  if (opt.brokerKey === 'coverage_hist') {
```

Comment:
What if another app developer wants to provide a data source that is exactly the same format but uses the name `coverage_histogram` instead of `coverage_hist`?

Reply:
Right, the OICR is going to have their own server. Now I get it what you mean before. 

I will switch to set up the property.